smartsim/settings/slurmSettings.py

```diff
@@ -201,7 +201,7 @@ def set_memory_per_node(self, memory_per_node: int) -> None:
         :param memory_per_node: Amount of memory per node in megabytes
         :type memory_per_node: int
         """
-        self.run_args["mem"] = f"{int(memory_per_node)}M"
+        self.run_args["mem"] = f"{str(int(memory_per_node))}M"
```

Comment:
Is the str cast needed here to satisfy the typechecker? Should be redundant as it is cast implicitly by the f-str

Reply:
doh!

Reply:
reverted these changes